test/framework/src/main/java/org/elasticsearch/script/MockScriptEngine.java

```diff
@@ -339,6 +343,9 @@ public void execute() {
                 }
             };
             return context.factoryClazz.cast(objectFieldScript);
+        } else if (context.instanceClazz.equals(DoubleValuesScript.class)) {
+            DoubleValuesScript.Factory doubleValuesScript = () -> new MockDoubleValuesScript();
```

Comment:
Can you add something that exercises this code?

Reply:
Sure

Reply:
I added a simple test, but also most of the tests in ScriptServiceTests will hit this code.